content/en/blog/_posts/2023-12-18-node-expand-secret-moves-ga

```diff
@@ -0,0 +1,94 @@
+---
+layout: blog
+title: "Kubernetes 1.29: CSI Storage Resizing Authenticated and Ready for GA in v1.29"
+date: 2023-12-18
+slug: csi-node-expand-secret-support-ga
+---
+**Authors:** Humble Chirammal, Louis Koo (deeproute.ai)
+
+In the dynamic universe of Kubernetes, version v1.29 unfolds a momentous chapter
+as support for authentication during CSI (Container Storage Interface) storage
+resize operations ascends from Beta to General Availability (GA).
+
+Let's embark on the evolution of this feature, initially introduced in alpha in
+Kubernetes v1.25, and unravel the profound changes accompanying its transition to GA.
```

Comment:
_profound_? I don't know the truth but I worry that readers will assume an LLM wrote this. Sorry.